sklearn/linear_model/_base.py

```diff
@@ -46,6 +46,31 @@
 # intercept oscillation.
 
 
+def _deprecate_normalize(normalize, default):
+    if normalize == 'deprecate':
+        _normalize = default
+    else:
+        _normalize = normalize
+
+    if default or (normalize != 'deprecate' and normalize):
+        warnings.warn(
+            "'normalize' was deprecated in version 0.24 and will be"
+            " removed in 0.26. If you wish to keep an equivalent"
+            " behaviour, use  Pipeline with a StandardScaler in a"
```

Comment:
This should be "If you wish to normalize"

because "an equivalent behaviour" will depend on what the user is doing